ui/containers/WorkspaceDetails/components/HiveDatabase.tsx

```diff
@@ -31,24 +32,31 @@ const HiveDatabase = ({ data, showModal, isPlatformOperations, isDefault }: Prop
   };
 
   return (
-    <Card>
+    <Card style={{ minHeight: 165 }}>
       <div style={{ fontSize: 14, overflow: 'hidden', textOverflow: 'ellipsis', textAlign: 'center' }}>{data.name}</div>
       <div style={{ display: 'flex', alignItems: 'center', justifyContent: 'center', marginTop: 12 }}>
-        <Doughnut
-          height={52}
-          width={52}
-          // @ts-ignore
-          data={sizeData}
-          redraw={false}
-          // @ts-ignore
-          options={{ legend: false, title: false, maintainAspectRatio: false }}
-        />
-      </div>
-      <div style={{ letterSpacing: 1, textAlign: 'center', fontSize: 12, padding: '4px 0 8px 0' }}>
-        {`${(allocated - consumed).toFixed(1)}GB/${allocated}GB`}
-        <br />
-        AVAILABLE
+        <Badge count={'Protocol: ' + protocol} style={{ backgroundColor: '#2D7493' }} />
       </div>
+      {protocol === 'hdfs' && (
```

Comment:
'hdfs' is hardcoded. You can make a Enum type, for those.
@afoerster  `protocol` can have several types, right?